### ISSUE TITLE
A11y_Visual Studio Code Client_Copilot_Copilot Inline Chat_Name Role Value: Incorrect name property is defined for the make request button as "Make request (Enter)" 

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#203774]
[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG4.1.2; #Name Role Value; #AILimited; 

### Environment and OS details:
Visual Studio Code insider version 1.86.0
Microsoft Windows 11 Enterprise (22H2)
OS Build 22621.3007

### Repro Steps:
1.Open Visual studio insider
2. In the Visual Studio Code Marketplace, go to the GitHub Copilot extension page and click Install.
3. Write a sample code and press CTRL+I to open the Copilot inline chat.
4. Observe that the Correct name property is defined for "Make request" button or not.

### Actual:
Incorrect name property is defined for the make request button as "Make request (Enter)" 

### Expected:
Correct name property should be defined as "Make request" button.

### User Impact:
Screen reader user will face difficulty if the incorrect name property information is being announced by screen reader

### Attachment:
![A11y_Visual studio Code Client_Copilot_Copilot_Inline chat_Name Role Value](https://github.com/microsoft/vscode/assets/98530069/faac57a8-19c2-46cb-8125-29ba97459227)

**Comment by meganrogge** [comment]
This is by design to make the keybinding of enter discoverable
